x/mongo/driver/topology/rtt_monitor.go

```diff
@@ -47,10 +50,10 @@ type rttMonitor struct {
 	connMu        sync.Mutex
 	samples       []time.Duration
```

Comment:
Now that we calculate `minRTT` using `movingMin`, can we remove `samples`?

Reply:
If we want to give the user the standard deviation of the RTTs, then we need to keep the samples slice.

Reply:
I don't think stddev is necessary anymore since we're using minimum RTT calculated from only 10 samples. See comment above.